eventActions/highlightActions.js

```diff
@@ -84,17 +84,22 @@ class highlightActions {
 	static async sendHighlightDM(client, user, message, highlightedPhrase) {
 		const workingMessage = message;
```

Comment:
Having worked with those code, do you know if there's any reason we need a second variable for the message? I know it's not your code, but while we're here, we might as well clean it up if we can.

Reply:
Huh I don't think that second variable is necessary, I'll test out the code without it. Thanks for the comment 👍